Vates/VatesAPI/src/vtkMDHexFactory.cpp

```diff
@@ -120,8 +117,7 @@ void vtkMDHexFactory::doCreate(
       API::IMDNode *box = boxes[i];
       Mantid::signal_t signal_normalized = (box->*normFunction)();
 
-      if (std::isfinite(signal_normalized) &&
-          m_thresholdRange->inRange(signal_normalized)) {
```

Comment:
Once this and PR #18300 are in master, I'll try parallelizing the for loops in `vtkMDHistoHexFactory` and (if successful) will add that to the release notes.